### DISCUSSION TITLE
Job offer - migrate Plausible to another server

### DISCUSSION THREAD

**Discussion by Tony-MF** [plausible/analytics#3580] [Self-Hosted Support]
Hello,

I need to migrate Plausible with about 300GB data (200GB compressed) to a different server and get it running. 

The best way to contact me is tony@mediafarmlimited.com, telegram -  @tony_mediafarm , or skype live:tony_19089. }

I can pay through Wise, Revolut or USDT (ERC20).

Thanks

**Comment by ruslandoga** [comment]
👋 @Tony-MF 

Some unfinished guides in https://github.com/plausible/hosting/tree/add-backup-restore-clickhouse-guide/guides might be useful :)

**Reply by ruslandoga** [reply]
🙏 And if you find a better way to backup ClickHouse, please open a PR or an issue in https://github.com/plausible/hosting with a short description of the steps performed!

**Reply by Tony-MF** [reply]
ChatGPT was telling me I can just move the docker volumes. Won't that work? https://chat.openai.com/share/61f50509-ffe5-41db-abc0-b12e538651a8

**Reply by ruslandoga** [reply]
That might work as well :) The new container would probably need to have the same UID/GID [(101 by default)](https://github.com/ClickHouse/ClickHouse/blob/b82d8479be54315a07562a21999f43f1b72bf67f/docker/server/Dockerfile.alpine#L57-L58), ~~ClickHouse users, roles, and databases.~~
